cyclus_input/predicting_the_past.xml

```diff
@@ -0,0 +1,183 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+
+<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
+  <control>
+    <duration>1020</duration>
+    <startmonth>1</startmonth>
+    <startyear>1965</startyear>
+  </control>
+  <archetypes>
+    <spec><lib>cycamore</lib> <name>Enrichment</name> </spec>
+    <spec><lib>cycamore</lib> <name>Reactor</name>    </spec>
+    <spec><lib>cycamore</lib> <name>Sink</name>       </spec>
+    <spec><lib>cycamore</lib> <name>Source</name>     </spec>
+    <spec><lib>cycamore</lib> <name>Storage</name>    </spec>
+    <spec><lib>cycamore</lib> <name>DeployInst</name> </spec>
+    <spec><lib>cycamore</lib> <name>ManagerInst</name></spec>
+    <spec><lib>agents</lib>   <name>NullRegion</name> </spec>
+  </archetypes>
+  <xi:include href="./buildtimes/buildtimes.xml#xpointer(/simulation/facilities/child::*)"/>
```

Comment:
I'm sorry if you've missed my previous comments. I should have renamed the simulation root tag to prevent confusion but the layers of tags, especially the presence of a root tag is to prevent the inclusion xml fragments, which prevent xml parsing.

I will divide these files into individual files so there isn't much navigation through tags but some files will need an arbitrary root tag to prevent xml fragments.